taf/tools/targets/__init__.py

```diff
@@ -237,3 +240,22 @@ def update_and_sign(path, library_dir, target_type, keystore, keys_description,
             click.echo()
             click.echo(str(e))
             click.echo()
+
+    @targets.command(context_settings=dict(
+        ignore_unknown_options=True,
+        allow_extra_args=True,
+    ))
+    @catch_cli_exception(handle=TAFError)
+    @click.option("--path", default=".", help="Authentication repository's location. If not specified, set to the current directory")
+    @click.option("--library-dir", default=None, help="Path to the library's root directory. Determined based on the authentication repository's path if not provided.")
+    @click.pass_context
+    def status(ctx, path, library_dir):
```

Comment:
Since this command is related to the repository as a whole, let's move it to the `repo` module/command.